tcping.go

```diff
@@ -526,21 +522,20 @@ func (tcpStats *stats) handleConnSuccess(rtt float32, now time.Time) {
 func tcping(tcpStats *stats) {
 	IPAndPort := netip.AddrPortFrom(tcpStats.ip, tcpStats.port)
 
-	connStart := getSystemTime()
-	conn, err := net.DialTimeout("tcp", IPAndPort.String(), oneSecond)
+	connStart := time.Now()
+	conn, err := net.DialTimeout("tcp", IPAndPort.String(), time.Second)
 	connEnd := time.Since(connStart)
 
 	rtt := nanoToMillisecond(connEnd.Nanoseconds())
-	now := getSystemTime()
 
 	if err != nil {
-		tcpStats.handleConnError(now)
+		tcpStats.handleConnError(time.Now())
 	} else {
-		tcpStats.handleConnSuccess(rtt, now)
+		tcpStats.handleConnSuccess(rtt, time.Now())
 		conn.Close()
 	}
 
-	time.Sleep(thousandMilliSecond - connEnd)
+	time.Sleep(time.Second - connEnd)
```

Comment:
@pouriyajamshidi why `- connEnd` btw? Won't it make requests go faster than expected (1 second)?

Reply:
We do not want to spend more than a second (roughly speaking) inside the `tcping` function.

Let's say a probe took _500 ms_ to complete (`connEnd`). We spend another _500 ms_ idling to complete the 1 second interval. Otherwise, we are too quick for the next probe.